cmd/relayfile-cli/relay_state_counter_test.go

```
@@ -0,0 +1,92 @@
+	stop := make(chan struct{})
+
+	// mountsync republishing public state, as the live daemon does ~1x/sec.
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		for {
+			select {
+			case <-stop:
+				return
+			default:
+				_ = syncer.FlushOutboxOnce(t.Context())
```

golangci-lint (2.12.2)

[error] stdversion: testing.Context requires go1.24 or later (module is go1.22)

(govet)